app/uk/gov/hmrc/helptosave/models/UCResponse.scala

```diff
@@ -16,10 +16,32 @@
 
 package uk.gov.hmrc.helptosave.models
 
-import play.api.libs.json.{Format, Json}
+import play.api.libs.json._
 
-case class UCResponse(ucClaimant: String, withinThreshold: Option[String])
+case class UCResponse(ucClaimant: Boolean, withinThreshold: Boolean)
```

Comment:
`withinThreshold` should be an `Option[Boolean]` - if it is not there we don't want to send that parameter to DES